frontend/playwright.config.ts

```diff
@@ -5,6 +5,7 @@ const url = `http://localhost:${port}`;
 
 export default defineConfig({
   testDir: "./tests",
+  //testMatch: "tests/lighthouse.spec.ts", //This runs light house test for only the lighhouse.spec.ts file
```

Comment:
Please take care to be consistent with the rest of the codebase, which uses lower-cased JS-doc style comments.